src/module-elasticsuite-indices/view/adminhtml/templates/ghostindices/toolbar.phtml

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteIndices
+ * @author    Vadym HONCHARUK <vahonc@smile.fr>
+ * @copyright 2025 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+?>
+
+<?php
+/** @var \Smile\ElasticsuiteIndices\Block\Adminhtml\GhostIndices\Toolbar $block */
+/** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+
+$ghostCount = $block->getGhostIndicesCount();
+
+if ($ghostCount === 1) {
+    $ghostMessage = __('1 ghost index found');
+} else {
+    $ghostMessage = __('%1 ghost indices found', $ghostCount);
+}
+
+$confirmationTitle = $block->escapeJs($block->escapeHtml(__('Are you sure?')));
+$confirmationMessage = $block->escapeJs($block->escapeHtml(__('This will permanently delete all Ghost indices. Continue?')));
```

Comment:
Please remove "permanently", to a non-native speaker, it might somehow tell the wrong message that no other ghost index will ever show up.

Reply:
Done.